src/components/MnemonicCreation.tsx

```diff
@@ -106,7 +106,7 @@ class MnemonicCreation extends React.Component<Props, State> {
                 returnKeyType="done"
                 multiline={true}
                 numberOfLines={3}
-                maxLength={80}
+                maxLength={100}
```

Comment:
Why was this changed?

Reply:
it was to short for most seed phrases :D

Reply:
Cool :)